### PR TITLE
online form builder page

### PR DESCRIPTION
**Pull request by dhaladitya108**

### CODE REVIEW

File: components/online-form-builder/ideal-form-builder.vue
Comment (piyushsinghania) [inline components/online-form-builder/ideal-form-builder.vue:1]: I see this component is very similar to `key-features.vue` component. Can you check if it is possible to get away with only one, while passing the data as props

**Review by piyushsinghania — APPROVED** [review]

File: pages/online-form-builder.vue [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+                'Do I need coding skills to use an online form builder? ',
+              answer:
+                'No, online form builders are designed to be user-friendly, eliminating the need for coding knowledge. You can easily drag and drop elements to create your desired form.',
+            },
+            {
+              question:
+                'How do I share my forms created with an online form builder? ',
+              answer:
+                'Most online form builders offer options to embed forms on websites, share links, or email the form to recipients.',
+            },
```
[the remark this answers is not in the document]
Comment (dhaladitya108) [reply pages/online-form-builder.vue:238]: yes it's a much better approach for showing the faqs 😀

File: components/online-form-builder/ideal-form-builder.vue
Comment (dhaladitya108) [reply components/online-form-builder/ideal-form-builder.vue:1]: I don't think so. There's styling varies a lot. 
image
image

File: pages/online-form-builder.vue [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+              position: 1,
+              name: 'Formester',
+              item: 'https://formester.com/',
+            },
+            {
+              '@type': 'ListItem',
+              position: 2,
+              name: 'Jotform101 - Complete Guide to Form Building by Formester',
+              item: 'https://formester.com/jotform101/',
+            },
```
[the remark this answers is not in the document]
Comment (dhaladitya108) [reply pages/online-form-builder.vue]: changed

File: pages/online-form-builder.vue [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+      ],
+    }
+  },
+  async asyncData() {
+    let randomTestimonials = await allTestimonials
+    const randIndex = Math.floor(
+      Math.random() * (randomTestimonials.length - 2)
+    )
+    randomTestimonials = randomTestimonials.slice(randIndex, randIndex + 2)
+    return { randomTestimonials }
```
[the remark this answers is not in the document]
Comment (dhaladitya108) [reply pages/online-form-builder.vue]: done

File: pages/online-form-builder.vue [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+  computed: {
+    meta() {
+      const metaData = {
+        type: 'website',
+        url: 'https://formester.com/online-form-builder/',
+        title: 'Online Form Builders for powerful form-building',
+        description:
+          'Effective data collection and management for businesses and individuals alike. Transform your data-collection process & derive valuable insights!',
+        mainImage: 'https://formester.com/jotform101-hero-section.png', // need to update with auto-responder page image
+        mainImageAlt: 'Form builder showing drag and drop functionality', // need to update with auto-responder page image alt
```
[the remark this answers is not in the document]
Comment (dhaladitya108) [reply pages/online-form-builder.vue]: I have changed it

File: assets/images/online-form-builder/ideal-form-builder/auto-responders.svg
[the remark this answers is not in the document]
Comment (dhaladitya108) [reply assets/images/online-form-builder/ideal-form-builder/auto-responders.svg:1]: done

File: assets/images/online-form-builder/ideal-form-builder/spam-protection.svg
[the remark this answers is not in the document]
Comment (dhaladitya108) [reply assets/images/online-form-builder/ideal-form-builder/spam-protection.svg:1]: done

**Pull request merged**